commit 1bfac90d1b8e63a4d44158c3445d8fda3fb6d5eb
From: Rigbert Hamisch <rigbert@gmx.de>
Date:   Tue Sep 27 10:46:43 2011 +0200
Subject: USB: qcserial: add device ID for "HP un2430 Mobile Broadband Module"
Patch-mainline: 3.2-rc1
References: bnc#726999

    USB: qcserial: add device ID for "HP un2430 Mobile Broadband Module"
    
    add device ID for "HP un2430 Mobile Broadband Module"
    
    Signed-off-by: Rigbert Hamisch <rigbert@gmx.de>
    Cc: stable <stable@kernel.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
Signed-off-by: Oliver Neukum <oneukum@suse.de>

diff --git a/drivers/usb/serial/qcserial.c b/drivers/usb/serial/qcserial.c
index d8651ea..b9bb247 100644
--- a/drivers/usb/serial/qcserial.c
+++ b/drivers/usb/serial/qcserial.c
@@ -28,6 +28,7 @@ static const struct usb_device_id id_table[] = {
 	{USB_DEVICE(0x05c6, 0x9212)},	/* Acer Gobi Modem Device */
 	{USB_DEVICE(0x03f0, 0x1f1d)},	/* HP un2400 Gobi Modem Device */
 	{USB_DEVICE(0x03f0, 0x201d)},	/* HP un2400 Gobi QDL Device */
+	{USB_DEVICE(0x03f0, 0x371d)},	/* HP un2430 Mobile Broadband Module */
 	{USB_DEVICE(0x04da, 0x250d)},	/* Panasonic Gobi Modem device */
 	{USB_DEVICE(0x04da, 0x250c)},	/* Panasonic Gobi QDL device */
 	{USB_DEVICE(0x413c, 0x8172)},	/* Dell Gobi Modem device */
